Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -1 +1,271 @@
 
+## Notes for Flutter TDD Clean Architecture Course Videos
+
+#### 1. Explanation & Project Structure
+> Video:https://www.youtube.com/watch?v=KjE2IDphA_U&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech
+
+In the `first tutorial` an explanation is given of the importance of having all functions tested in our application.  
+It also explains the **clean architecture**.
+All the features of the application will be divided into three folders, **domain**, **data** and **presentation**.
+It is also explained how each of these folders works and what they will contain.
+
+<br />
+
+#### 2. Entities & Use Cases 
+> Video:https://www.youtube.com/watch?v=lPkWX8xFthE&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech&index=2
+
+In the `second tutorial` we start by adding the packages of the dependencies that will be needed to build the application.
```

Comment:
what _file_ do the `dependecies` get added to?

Reply:
We add the dependencies to the pubspec.yaml file.

Reply:
Miguel, answering the question in a comment does not _magically_ transfer the information into the Markdown file. 
I _know_ the `dependencies` are added to the `pubspec.yaml` file, but the people _reading_ this file don't. 
The purpose of review comments is not for _us_ to have a casual conversation, it's to _improve_ the content.

Reply:
I had already added that part to the file.


![Screen Shot 2020-02-18 at 10 13 32](https://user-images.githubusercontent.com/27420533/74726479-6df9fc80-5237-11ea-90f6-c68bf2e5c089.png)

Reply:
Thanks. 👍